src/Toolkit/Toolkit/UI/Controls/PopupViewer/TextPopupElementView.Maui.cs

```diff
@@ -455,6 +451,16 @@ private static bool MapsToBlock(MarkupNode node)
         {
             return node.Type is MarkupType.List or MarkupType.Table or MarkupType.Block or MarkupType.Divider or MarkupType.Image;
         }
+
+        private PopupViewer? GetPopupViewerParent()
+        {
+            var parent = this.Parent;
+            while (parent is not null && parent is not PopupViewer popup)
+            {
+                parent = parent.Parent;
+            }
+            return parent as PopupViewer;
+        }
```

Comment:
Move to Popupviewer and make static. See https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/ac1bc3e6c5a6239f60e9f9dc42f986f3495ade3b/src/Toolkit/Toolkit/UI/Controls/FeatureForm/FeatureFormView.Maui.cs#L161 for example